api/models/Categories.js

```diff
@@ -0,0 +1,18 @@
+// Require Mongose ORM 
+var Mongoose = require('mongoose');
+// Require Mongoose Schema to Make Mongoose Object
+var Schema = Mongoose.Schema;
+
+// Lets create Schema Object
+// The formet will be x = { variables like type and conditionals...} fallowed by ','
+var CategorySchema = new Schema({
```

Comment:
in my head, it made sense to make a category and tag set of documents? But like @luoto said, maybe we should just keep all references to this within the Project documents? What do you think @ilvcs , @luoto

Reply:
can we do it like this?
category: {
 {categoryName: 'web app', projectsById: ['1234', '4567', '1421321321']},
{categoryName: 'mobileapp', projectsById: ['1234', '4567', '1421321321']},
}

Reply:
cat- _id1: [node,react]
       _id2:[mobile, reactNative]
       _id3:[ python,react]
cat.get(key: react and node ) -> {id1,id3}

Reply:
I think we should include it in the projects model. Here we are only storing the category name anyway. From my understanding of NoSQL collections is that it is designed around normalization, but here we are designing the collections here like a SQL database.